.github/.OwlBot.yaml

```diff
@@ -4,6 +4,7 @@ docker:
 deep-remove-regex:
   - /accessapproval/apiv1/
   - /accesscontextmanager/apiv1/
+  - /ai/generativelanguage/v1beta2
```

Comment:
Looks like for this in the value below you should add a trailing slash.

Reply:
Good catch, thank you!